airtime_mvc/public/js/airtime/library/podcast.js

```diff
@@ -436,51 +536,114 @@ var AIRTIME = (function (AIRTIME) {
      *
-     * @param {Object}  podcast   the podcast data JSON object.
-     * @param {Tab}     tab       Tab object the podcast will be opened in
+     * @param {jQuery}  domNode   the jQuery DOM node to create the table inside.
      * @param {Object}  params    JSON object containing datatables parameters to override
      * @param {Object}  buttons   JSON object containing datatables button parameters
+     * @param {Object}  config    JSON object containing internal PodcastEpisodeTable parameters
+     * @param {boolean} config.hideIngestCheckboxes flag denoting whether or not to hide checkboxes for ingested items
      *
-     * @returns {*} the created Table object
+     * @returns {Table} the created Table object
      */
-    mod.initPodcastEpisodeDatatable = function(podcast, tab, params, buttons) {
-        buttons = $.extend({
-            slideToggle: {
-                title           : '',
-                iconClass       : 'spl-no-r-margin icon-chevron-up',
-                extraBtnClass   : 'toggle-editor-form',
-                elementId       : '',
-                eventHandlers   : {}
-            }
-        }, buttons);
-        params = $.extend(params,
+    mod.initPodcastEpisodeDatatable = function (domNode, params, buttons, config) {
+        if ('slideToggle' in buttons) {
+            buttons = $.extend(true, {
+                slideToggle: {
+                    title: '',
+                    iconClass: 'spl-no-r-margin icon-chevron-up',
+                    extraBtnClass: 'toggle-editor-form',
+                    elementId: '',
+                    eventHandlers: {},
+                    validateConstraints: function () { return true; }
+                }
+            }, buttons);
+        }
+        params = $.extend(true, params,
             {
                 oColVis: {
-                    sAlign      : 'right',
-                    aiExclude   : [0, 1],
-                    buttonText  : $.i18n._("Columns"),
-                    iOverlayFade: 0,
-                    oColReorder : {
+                    aiExclude: [0, 1],
+                    oColReorder: {
                         iFixedColumns: 1  // Checkbox
                     }
+                },
+                fnCreatedRow: function(nRow, aData, iDataIndex) {
+                    var self = this;
+                    $(nRow).draggable({
+                        helper: function () {
+                            var $row = $(this), data = self._datatable.fnGetData(nRow);
+                            $row.data("aData", data.file);
+                            self.selectRow(this, data, self.SELECTION_MODE.SINGLE, $row.index());
+                            var selected = self.getSelectedRows().length, container,
+                                width = self._$wrapperDOMNode.closest(".dataTables_wrapper").outerWidth(), message;
+
+                            message = sprintf($.i18n._("Adding %s Item%s"), selected, selected > 1 ? "s" : "");
```

Comment:
This way of doing pluralization doesn't work for some languages. Pluralization is built into the jQuery localization plugin we're using. Search this page for "plural" to see the right way to do this:
https://github.com/wikimedia/jquery.i18n